.github/workflows/changelog_check.yml

```diff
@@ -0,0 +1,16 @@
+name: Changelog
+on: [pull_request]
+
+jobs:
+  check-newsfile:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: ${{github.event.pull_request.head.sha}}
```

Comment:
Looks like equivalent data is available as `${{ github.head_ref }}`... might as well use that to be more obvious.